birdhouse/components/proxy/conf.d/frontend.conf.template

```diff
@@ -27,22 +27,4 @@ server {
     }
 }
 
-server {
-    listen       443;
-    server_name  localhost;
-    proxy_buffering off;
-
-    resolver 127.0.0.11;
-
-    ssl    on;
-    ssl_certificate     /etc/nginx/cert.pem;
-    ssl_certificate_key /etc/nginx/cert.pem;
-
-    include /etc/nginx/conf.d/all-services.include;
-
-    # redirect server error pages to the static page /50x.html
-    error_page   500 502 503 504  /50x.html;
-    location = /50x.html {
-        root   /usr/share/nginx/html;
-    }
-}
+${PROXY_INCLUDE_HTTPS}
```

Comment:
See other comment in proxy `default.env` for more context.

I think it is better to leave it there regardless and let the HTTPS redirection or not deal with which "server" config gets hit by the requests.

Reply:
See comment here: https://github.com/bird-house/birdhouse-deploy/pull/484#discussion_r1871460056

Specifically: 

>> but it seems the https.include rewrite is not needed (the HTTPS server definition is just ignored).

> It's needed if you don't have a self signed SSL certificate. Otherwise the proxy container won't start since nginx fails to parse a valid SSL certificate in the https block.